### PR TITLE
Wheel building/publishing

### PR DESCRIPTION
**Pull request by kwilcox**
Checks off `artifact publication` of https://github.com/ioos/ioos_qc/issues/55... if it works.

### CODE REVIEW

File: .github/workflows/publish.yml
Comment (ocefpaf) [inline .github/workflows/publish.yml:18]: It is probably safe to do a `'3.x'` here. It would ensure that any Python 3 available in the image could be used.

File: .github/workflows/publish.yml
Comment (ocefpaf) [inline .github/workflows/publish.yml:47]: Someone with a PyPI account and onwership to ioos_qc should create the token.
Actually is ioos_qc on PyPI?

File: .github/workflows/publish.yml
Comment (ocefpaf) [inline .github/workflows/publish.yml:7]: @kwilcox this LGTM but if you want to test it in the PR before merging you can change lines 4-7 to:

```yaml
on:
  pull_request:
```

so it runs here. We will need to change it back to releases only before merging.

**Review by ocefpaf — APPROVED** [review]

File: .github/workflows/publish.yml
Comment (kwilcox) [reply .github/workflows/publish.yml:47]: https://pypi.org/project/ioos-qc/

I will add the secret

Comment (kwilcox) [reply .github/workflows/publish.yml:50]: @mwengren can you make me a repository owner so I may add secrets? Alternatively I can send you the token to add via other secure methods, lmk!

File: conda-recipe/meta.yaml
Comment (ocefpaf) [inline conda-recipe/meta.yaml:3]: Since you are building off GitHub you can do:

```yaml
{% set sdata = load_setup_py_data() %}

package:
  name: panel
  version: {{ sdata['version'] }}

source:
  path: ..

build:
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
  entry_points:
    {% for group,epoints in sdata.get("entry_points",{}).items() %}
    {% for entry_point in epoints %}
    - {{ entry_point }}
    {% endfor %}
    {% endfor %}

requirements:
  build:
    - python {{ sdata['python_requires'] }}
    {% for dep in sdata['extras_require']['build'] %}
    - {{ dep }}
    {% endfor %}
  run:
    - python {{ sdata['python_requires'] }}
    {% for dep in sdata.get('install_requires',{}) %}
    - {{ dep }}
    {% endfor %}

about:
  home: {{ sdata['url'] }}
  summary: {{ sdata['description'] }}
  license: {{ sdata['license'] }}
```

File: .github/workflows/publish.yml
Comment (mwengren) [reply .github/workflows/publish.yml:50]: @kwilcox Sorry, sometimes GitHub traffic gets skipped over (by me, by accident).  I just gave you permissions.  LMK when this is working and I will change you back because we need to keep Admin privileges as restricted as possible.

File: conda-recipe/meta.yaml
Comment (kwilcox) [reply conda-recipe/meta.yaml:3]: will leave this for a later date, I still build it locally :grimacing:

File: .github/workflows/publish.yml
Comment (mwengren) [reply .github/workflows/publish.yml]: @kwilcox Are we good with this now?  Will reset permissions if so.

Comment (kwilcox) [reply .github/workflows/publish.yml]: Yes, i added the key, all set

**Pull request merged**
